frontend/src/theming/light.less

```diff
@@ -1,5 +1,6 @@
 body.light {
   @import '~antd/dist/antd.less';
+  @import 'antd-button-color/dist/css/style.less';
```

Comment:
```suggestion
```
This shouldn't be required. Doesn't it work correctly if you remove it?

Reply:
unfortunately this line is required both for dark and light theme to make `antd-button-color` work